src/components/study-pane.js

```diff
@@ -1243,7 +1417,7 @@ const StudyPane = (props) => {
                                 display: 'flex',
                                 pointerEvents: 'none',
                                 flexDirection: 'column',
-                                marginLeft: drawerOpen ? 0 : drawerToolbarWidth,
```

Comment:
You must keep a marginLeft for the single Line Diagram for drawerToolbarWidth nope ?

Reply:
No, no need for margin left.

Reply:
because the margin of the drawer was corrected (could have been part of a separate PR) 
```
-        marginLeft: -drawerWidth - drawerToolbarWidth,
+        marginLeft: -drawerExplorerWidth,
```